docs/src/examples/components/List/Types/ListExampleActive.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { List, Image } from '@stardust-ui/react'
+
+const ListExampleActive = ({ knobs }) => (
+  <List>
+    <List.Item
+      media={<Image src="public/images/avatar/small/matt.jpg" avatar />}
```

Comment:
note that you need to provide keys here as well - otherwise we will have the following errors/warnings in the console: 

![image](https://user-images.githubusercontent.com/9024564/46951339-f8e94680-d087-11e8-88cb-260fe04f2a03.png)